### PR TITLE
Improve list of default namespaces

### PR DESCRIPTION
**Pull request by sftim**
- Move the list of default namespaces earlier in https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/  [[preview](https://deploy-preview-37739--kubernetes-io-main-staging.netlify.app/docs/concepts/overview/working-with-objects/namespaces/)]
- Sort that alphabetically
- Use a description list

### CODE REVIEW

File: content/en/docs/concepts/overview/working-with-objects/namespaces.md [hunk 1]
```diff
@@ -32,6 +32,26 @@ resources, such as different versions of the same software: use
...
+{{< note >}}
+For a production cluster, consider _not_ using the `default` namespace. Instead, make other namespaces and use those.
+{{< /note >}}
+
+## Initial namespaces
+
+Kubernetes starts with four initial namespaces:
+
+`default`
+: Kubernetes includes this namespace so that you can start using your new cluster without first creating a namespace.
```
Comment (chrisnegus) [inline content/en/docs/concepts/overview/working-with-objects/namespaces.md:44]: May be implied, but you could say that any Pods deployed without an identified namespace automatically go into the `default` namespace.

**Review by chrisnegus — COMMENTED** [review]
Looks good. I have one suggestion.

File: content/en/docs/concepts/overview/working-with-objects/namespaces.md (hunk 1 above)
Comment (chrisnegus) [reply content/en/docs/concepts/overview/working-with-objects/namespaces.md:44]: Fair enough.

**Review by divya-mohan0209 — COMMENTED** [review]
/approve

**Pull request merged**